lib/active_interaction/base.rb

```diff
@@ -36,6 +36,20 @@ class << self
       include Hashable
       include Missable
 
+      ALLOWED_KWARGS = %i[
+        desc
+        default
+        index_errors
+        strip
+        format
+        digits
+        base
+        from
+        methods
+        class
+        converter
+      ].freeze
```

Comment:
These were all the options I could find from the those mentioned in the docs.

Not all of these options are possible for every kind of filter; a more sophisticated version of this would verify that the given option was possible for the filter, but I wanted to stand up a PR to see what people think first.